Stratis.Bitcoin/MemoryPool/MempoolManager.cs

```diff
@@ -48,19 +52,51 @@ public List<TxMempoolInfo> InfoAll()
 			{
 				Trx = item.Transaction,
 				Time = item.Time,
-				FeeRate = new FeeRate(item.Fee, (int) item.GetTxSize()),
+				FeeRate = new FeeRate(item.Fee, (int)item.GetTxSize()),
 				FeeDelta = item.ModifiedFee - item.Fee
 			}).ToList();
 		}
 
+		internal async Task LoadPool(string fileName = null)
```

Comment:
This happens in AcceptToMemoryPoolWorker with CheckMempoolCoinView, which is called from AcceptToMemoryPoolWithTime.

The concept here is to pass the memory pool entries through the same validation which occurs when receiving them from a node.

There generally shouldn't be any conflicts, because the mempool.dat file is saved when the node is shutting down, and is loaded when it starts back up; but in the case that someone takes an old memory pool and loads it on a node which has a later blockchain, the result should be the same as if a node was sending outdated transactions.